drivers/storage/portworx/manifest/manifest.go

```diff
@@ -29,9 +29,12 @@ const (
 	defaultAutopilotImage      = "portworx/autopilot:1.3.2.1"
 	defaultLighthouseImage     = "portworx/px-lighthouse:2.0.7"
 	defaultNodeWiperImage      = "portworx/px-node-wiper:2.10.0"
-	defaultTelemetryImage      = "purestorage/ccm-service:3.2.11"
+	defaultCCMJavaImage        = "purestorage/ccm-service:3.2.11"
 	defaultCollectorProxyImage = "envoyproxy/envoy:v1.21.4"
 	defaultCollectorImage      = "purestorage/realtime-metrics:1.0.1"
+	defaultCCMGoImage          = "pureezhang/ccm-go:0.0.8"
+	defaultLogUploaderImage    = "pureezhang/log-upload:0.0.11"
```

Comment:
Why are these still referencing Eugene's private docker repo? This should now come from our official docker hub. Please work with Eugene to upload and use that.